.github/workflows/jira-issue-create.yml

```diff
@@ -0,0 +1,33 @@
+name: Jira Issue Creator
+
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    environment: Jira
+    name: SDK Bot Jira Issue Creation
+    steps:
+      - name: Login
+        uses: atlassian/gajira-login@master
+        env:
+          JIRA_BASE_URL: ${{ secrets.JIRA_BASE_URL }}
```

Comment:
These names are a little different than the ones provided in the documentation [here](https://github.com/marketplace/actions/jira-issue-creation), is that OK?

Reply:
Oh nevermind, the keys are fine, as listed here:
https://github.com/marketplace/actions/jira-login

Reply:
Wrong Action, we used the official [Atlassian one](https://github.com/marketplace/actions/jira-create-issue)